app/models/service_binding.rb

```diff
@@ -28,34 +28,55 @@ def create(app_id)
 
     raise RoleAlreadyCreated.new("Host identity already exists.") if host.exists?
 
-    result = load_policy(template_create)
-    host =
+    host_id =
       if ConjurClient.policy != 'root'
-        "host/#{ConjurClient.policy}/#{@binding_id}"
+        "#{ConjurClient.policy}/#{@binding_id}"
       else
-        "host/#{@binding_id}"
+        @binding_id
       end
     
+    api_key = (ConjurClient.version == 4 ? create_v4(host_id) : create_v5)
```

Comment:
`create` to me doesn't imply that this method returns the api key after creating - can you update the name of these methods to make this clearer?

Reply:
Hmm, any suggestions for a better name? The method is creating a host so I'm not sure what could be better. I don't think it's totally bizarre to return an api key from a host creation operation but I'm open to ideas!